syntax = "proto3";

package varlog;

import "github.com/gogo/protobuf/gogoproto/gogo.proto";

option go_package = "github.daumkakao.com/varlog/varlog/proto/sequencer";

message SequencerRequest {
        uint64 epoch = 1;
}

message SequencerResponse {
        uint64 glsn = 1;
}

service SequencerService {
        rpc Next(SequencerRequest) returns (SequencerResponse) {}
}
